src-tauri/Cargo.toml

```diff
@@ -71,3 +70,7 @@ codegen-units = 1
 # TODO: Remove once chacha20poly1305 0.10 appears in libp2p's dependencies
 chacha20poly1305 = { git = "https://github.com/RustCrypto/AEADs", rev = "06dbfb5571687fd1bbe9d3c9b2193a1ba17f8e99" }
 libp2p = { git = "https://github.com/subspace/rust-libp2p", branch = "subspace-v3" }
+tracing = { git = "https://github.com/ozgunozerk/tracing-keep-last-n-logs", branch = "v0.1.x" }
+tracing-core = { git = "https://github.com/ozgunozerk/tracing-keep-last-n-logs", branch = "v0.1.x" }
+tracing-appender = { git = "https://github.com/ozgunozerk/tracing-keep-last-n-logs", branch = "v0.1.x" }
+tracing-subscriber = { git = "https://github.com/ozgunozerk/tracing-keep-last-n-logs", branch = "v0.1.x" }
```

Comment:
I'd leave a comment on why this is necessary and move forks into our repo, so we can keep it forever and make sure we can build corresponding commits in the future.